source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,320 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern is an Object Oriented design technique for creating complex
+immutable objects separately from their representation. The builders provided
+by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to use the builder pattern
+to specify sort criteria for your queries. If you are new to sorting in the
+MongoDB Java driver, consider reading our
+:doc:`CRUD Sort Guide </fundamentals/crud/read-operations/sort/>`
```

Comment:
Let's omit CRUD since the reader might not understand the context immediately.

Reply:
"Sort Guide", another way "Sort guide in the fundamentals section", "guide on sorting",  it should be descriptive makes sense in the context of the sentence.

Reply:
Thanks for catching this. Fix: https://github.com/mongodb/docs-java/pull/59/commits/b553ec5054687057f28fa28a12654d177e186654

Going forward, I will try to give links immediately understandable context, leaving out unhelpful/jargon words when possible.